Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -11278,8 +11286,6 @@ void WebPageProxy::resetStateAfterProcessExited(ProcessTerminationReason termina
         protectedPageLoadState->reset(transaction);
     }
 
-    updatePlayingMediaDidChange(MediaProducer::IsNotPlaying);
```

Comment:
I removed this because it seemed like it was doing nothing--we are already calling it in `resetState` above.